APMrover2/mode_steering.cpp

```diff
@@ -15,38 +15,41 @@ void ModeSteering::update()
     float desired_steering, desired_speed;
     get_pilot_desired_steering_and_speed(desired_steering, desired_speed);
 
+    bool reversed = is_negative(desired_steering);
```

Comment:
Is this really supposed to be `desired_steering` and not `desired_speed`?

Reply:
That was definitely a bug, fixed in follow up commit.